gradle/mvn-push-android.gradle

```diff
@@ -97,6 +118,60 @@ afterEvaluate { project ->
                             name POM_DEVELOPER_NAME
                         }
                     }
+
+                    if (POM_DESCRIPTION == "Mapbox Android Telemetry Library") {
+                        String okhttpString = dependenciesList.okhttp3
+                        def okhttp = okhttpString.split(':')
+
+                        String gsonString = dependenciesList.gson
+                        def gson = gsonString.split(':')
+
+                        String annotationString = dependenciesList.supportAnnotation
+                        def supportAnnotation = annotationString.split(':')
+
+                        def dependenciesArray = [okhttp, gson, supportAnnotation]
+                        def exclusionsArray = []
+
+                        if (!VERSION_NAME.contains('LITE')) {
+                            String coreString = "com.mapbox.mapboxsdk:mapbox-android-core:" + core.version
+                            def coreArray = coreString.split(':')
+                            dependenciesArray.add(coreArray)
+
+                            String lifecycleString = dependenciesList.archLifecycleExtensions
+                            def lifecycleArray = lifecycleString.split(':')
+                            dependenciesArray.add(lifecycleArray)
+
+                            def viewModel = ["viewmodel", "android.arch.lifecycle"]
```

Comment:
why are we hardcoding all these? are we expected to hardcode every time we add/remove dependencies?